### PR TITLE
DEVHUB-920: Migrate Tests to Strapi-Only

### PR DESCRIPTION
**Pull request by jestapinski**
## Links:

-   [JIRA Ticket](https://jira.mongodb.org/browse/DEVHUB-920)

## Description:

-   This PR adjusts our CI tests to only consume testing data from the CMS post-test data import.

## Testing

-   The automated tests should work.

## What types of outside events need to happen for this PR?

-   [ ] Documentation Updates
-   [ ] Product Review
-   [ ] Design Review
-   [ ] Release Note Update (only for deployments)

### CODE REVIEW

**Comment by jestapinski** [comment]
my goodness it worked, feel free to approve, merge into your branches when ready

File: cypress/integration/learn.js [hunk 1]
```diff
@@ -1,8 +1,8 @@
 const CANONICAL_URL = 'https://www.mongodb.com/developer/learn/';
 const FIRST_ARTICLE_IN_ORDERING = '/how-to/transactions-c-dotnet/';
-const FIRST_ARTICLE_UPDATED_DATE = 'Oct 03, 2020';
+const FIRST_ARTICLE_UPDATED_DATE = 'Nov 18, 2025';
```
Comment (harikakotipalli) [inline cypress/integration/learn.js:3]: the date seem incorrect to me. It should reflect the strapi article's first updated date isn't it?

Comment (harikakotipalli) [reply cypress/integration/learn.js:3]: Also one clarification - STRAPI_PUBLICATION_STATE=live is set to true by default correct, so we are only pulling cms-test articles that are published for running these tests, correct?

**Comment by jestapinski** [comment]
Okay @harikakotipalli tests still passing here

**Review by harikakotipalli — APPROVED** [review]

**Comment by jestapinski** [comment]
Thanks!

**Pull request merged**